src/Compiler/Microsoft.AspNetCore.Razor.Language/test/TestFiles/IntegrationTests/ComponentDesignTimeCodeGenerationTest/CascadingGenericInference_ParameterInNamespace/TestComponent.ir.txt

```diff
@@ -1,5 +1,5 @@
 ﻿Document - 
-    NamespaceDeclaration - (11:0,11 [18] x:\dir\subdir\Test\TestComponent.cshtml) - MyApp.Components
+    NamespaceDeclaration - (11:0,11 [16] x:\dir\subdir\Test\TestComponent.cshtml) - MyApp.Components
```

Comment:
Note with the latest change this is actually a design time update too, but its so minor I'm not worried about it causing issues with design time docs and so doesn't seem worth trying to plumb stuff through to keep the original values.